packages/content-type-reply/README.md

```diff
@@ -2,6 +2,9 @@
 
 This package provides an XMTP content type to support direct replies to messages.
 
+> **Open for feedback**  
+> You are welcome to provide feedback on this implementation by commenting on the [Reply Content Type XIP](https://github.com/xmtp/XIPs/pull/22) (XMTP Improvement Proposal).
```

Comment:
I think we are actually encouraging people to comment in the discussion and not the XIP. https://github.com/orgs/xmtp/discussions/35